project/src/main/webapp/scripts/editQuestionOutlinesScripts.js

```diff
@@ -87,6 +87,7 @@ function nextButton() {
   }
 }
 
+var lx, ly, rx, ry;
```

Comment:
Although I can understand it try using some more descriptive variable names so we can follow the flow a little easier.

i.e topLeftXCoord or something similar

Reply:
np -- implemented